Makefile

```diff
@@ -121,7 +121,21 @@ e2e-test: ## Run e2e tests against cluster pointed to by ~/.kube/config
 		-count 1 \
 		-timeout 90m \
 		-v \
-		./suites/... \
+		./suites/integration/... \
+		--ginkgo.focus="${FOCUS}" \
+		--ginkgo.timeout=90m \
+		--ginkgo.v
+
+.PHONY: sharing-test
+sharing-test: ## Run e2e tests against cluster pointed to by ~/.kube/config and using secondary account role. Cluster VPC must not be associated to a Service Network.
+	@kubectl create namespace $(e2e-test-namespace) > /dev/null 2>&1 || true # ignore already exists error
+	LOG_LEVEL=debug
+	cd test && go test \
+		-p 1 \
+		-count 1 \
+		-timeout 90m \
+		-v \
+		./suites/sharing/... \
 		--ginkgo.focus="${FOCUS}" \
 		--ginkgo.timeout=90m \
 		--ginkgo.v
```

Comment:
please dont copy paste code

Reply:
What the specific reason we need a separate `./suites/sharing/` folder?

 Are we able to keep the new test file still in the `./suites/integration` and in the `developer.md` say:
to run the RAM share test, pass the FOCUS env variable? no?
```
FOCUS=RAM REGION=us-west-2 SECONDARY_ACCOUNT_TEST_ROLE_ARN=arn:aws:iam::000000000000:role/MyRole make sharing-test
```

Reply:
They require different environments due to needing the cluster VPC to have no service network association, plus the use of a secondary account. In the future, we may even want to test multi-cluster scenarios (2, maybe 3 accounts with their own clusters). Putting these tests in the old suite would be confusing, and if we are going to isolate them with FOCUS, then we may as well isolate them as separate suites.

Reply:
Will look into skipping the RAM tests by default with ginkgo filtering specs. Then, if we pass a command to run the sharing tests, we run just that test.